src/pages/index.js

```diff
@@ -6,121 +6,415 @@ import Section from "../components/Section.js";
+//const addNewCardModal = document.querySelector("#add-card-modal");
 const addCardForm = document.querySelector("#add-card-form");
+// const addNewCardSubmitButton = addNewCardModal.querySelector(".modal__button");
+
+
+
+const cardsListElement = document.querySelector(".cards__list");
+
+
+
 
 //CREATE NEW INSTANCES OF ALL CLASSES // INITIALIZE
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "7bed9883-2658-4b5e-974b-2c2e17824760",
+    "Content-Type": "application/json",
+  },
+});
+
+const deleteCardPopup = new PopupWithDeleteConfirmation(
+  "#delete-card-modal",
+  handleConfirmDeleteSubmit,
+);
+
+deleteCardPopup.setEventListeners();
+
 const cardPreviewPopup = new PopupWithImage("#preview-image-modal");
 //cardPreviewPopup.setEventListeners();
 
-const editProfileFormValidator = new FormValidator(config, editProfileForm);
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
-);
-cardSection.renderItems();
+
+const editProfileFormValidator = new FormValidator(config, editProfileForm);
 
 const editProfilePopup = new PopupWithForm(
   selectors.editProfileModal,
   handleEditProfileFormSubmit
 );
-//editProfilePopup.setEventListeners();
-
-console.log(editProfileModal);
 
 const editUserInfo = new UserInfo({
   titleSelector: ".profile__title",
   occupationSelector: ".profile__description",
+  avatarSelector: ".profile__image",
 });
 
 const addNewCardPopup = new PopupWithForm(
   selectors.addNewCardModal,
   handleAddCardFormSubmit
 );
-console.log(addNewCardModal);
+
+const updateAvatarPopup = new PopupWithForm(
+  selectors.updateAvatarModal, 
+  handleProfileUpdateAvatarFormSubmit
+);
 
 const addNewCardFormValidator = new FormValidator(config, addCardForm);
 addNewCardFormValidator.enableValidation();
 
+const updateAvatarFormValidator = new FormValidator(config, updateAvatarForm);
+updateAvatarFormValidator.enableValidation();
+
+
+//USERS
+
+//API instances
+
+//User routes
+
+// GET /users/me – Get the current user’s info
+// PATCH /users/me – Update your profile information
+// PATCH /users/me/avatar – Update avatar
+// Card routes
+
+// GET /cards – Get all cards
+// POST /cards – Create a card
+// DELETE /cards/:cardId – Delete a card
+// PUT /cards/:cardId/likes – Like a card
+// DELETE /cards/:cardId/likes – Dislike a card
+
+
+//User Profile 
+
+let userInfo;
+
+
+// TODO delete
+
+api
+  .getUserInfo() //fetch info using api.getUserInfo()
+  .then((result) => {
+    userInfo = result; //store info in const
+    console.log(userInfo);
+    return api.getInitialCards();
+  })
+  .then((initialCards) => {
+    console.log(initialCards);
+    // //console.log(cardData);
+    // return userInfo._id;
+  })
+  // .then((card) => {
+  //   console.log(card);
+  //   console.log()
+  // })
+  .catch((err) => {
+    console.error(err); 
+    console.error("")
```

Comment:
`    console.error("")
`

It's not needed because you have `console.error(err); ` in line 156